calibration/PARADK/CGINS-PARADK-20447__20220131.csv

```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20447,CC_dark_offset,1.43,source_file PARAD-K_QSP2200_SN_20447_Calibration_2022-01-31.pdf
+20447,CC_scale_wet,9.01E-18,
```

Comment:
good, with CC_dark_offset Probe Dark and CC_scale_wet units Volts/(quanta/cm^2sec)